### PR TITLE
DOC-4313: General info about onlyoffice add-on

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: docs/OnlyOffice.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.. _OnlyOffice:
+
+#############################
+OnlyOffice Connector by eXo
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
OnlyOffice Connector for eXo
```

File: docs/OnlyOffice.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#############################
+
+**What is OnlyOffice**
+
+`OnlyOffice <https://www.onlyoffice.com/>`__ is a *partner editor service*
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
`OnlyOffice Online Editors <https://www.onlyoffice.com/office-suite.aspx>`__ is a great open source solution 
```

File: docs/OnlyOffice.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+**What is OnlyOffice**
+
+`OnlyOffice <https://www.onlyoffice.com/>`__ is a *partner editor service* 
+that allows you to create, edit  and collaborate online on your office 
+documents.
+
+|image0|
+
+.. note:: The supported documents types are: docx, pptx and xlsx.
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
.. note:: OnlyOffice is 100% compatible with the following Microsoft Office document formats : docx, pptx and xlsx.
```

File: docs/OnlyOffice.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+eXo OnlyOffice connector add-on is available only with the **commercial
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
eXo partnered with OnlyOffice to offer the enterprise-ready version of OnlyOffice editors to its customers along with the **commercial
```

File: docs/OnlyOffice.rst [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+`OnlyOffice <https://www.onlyoffice.com/>`__ is a *partner editor service* 
+that allows you to create, edit  and collaborate online on your office 
+documents.
+
+|image0|
+
+.. note:: The supported documents types are: docx, pptx and xlsx.
+
+eXo OnlyOffice connector add-on is available only with the **commercial 
+editions of eXo Platform**. In addition, it requires the purchase of a dedicated
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
editions of eXo Platform**. Thus, eXo customers are required to purchase a
```

File: docs/OnlyOffice.rst [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+add-on subscription.
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
dedicated subscription to run the OnlyOffice connector add-on on their eXo Platform instance.
```

File: docs/OnlyOffice.rst [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+Having the above buisiness conditions satisfied, you can install the add-on using this command:
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
However, a 30-day trial is prepackaged. You can test-drive the connector by installing the add-on using this command:
```

**Review by plamarque — CHANGES_REQUESTED** [review]
various enhancements suggested. see comments.

File: docs/OnlyOffice.rst (hunk 4 above)
Comment (mzorai) [reply docs/OnlyOffice.rst]: Looks like a typing error I will change the second eXo by OnlyOffice online editors

File: docs/OnlyOffice.rst [hunk 8]
```diff
@@ -0,0 +1,29 @@
+.. _OnlyOffice:
+
+#############################
+OnlyOffice Connector for eXo
+#############################
+
+**What is OnlyOffice**
+
+`OnlyOffice Online Editors <https://www.onlyoffice.com/office-suite.aspx>`__ is a great open source solution 
+that allows you to create, edit  and collaborate online on your office
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
that lets you to create, edit  and collaborate online on your office 
```

File: docs/OnlyOffice.rst (hunk 4 above)
Comment (plamarque) [reply docs/OnlyOffice.rst]: yes, my bad

Comment (plamarque) [reply docs/OnlyOffice.rst]: fixed and slightly reworked

**Review by plamarque — CHANGES_REQUESTED** [review]
a few more language suggestions

File: docs/OnlyOffice.rst [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+**What is OnlyOffice**
+
+`OnlyOffice Online Editors <https://www.onlyoffice.com/office-suite.aspx>`__ is a great open source solution 
+that lets you to create, edit  and collaborate online on your office 
+documents.
+
+|image0|
+
+.. note:: OnlyOffice is 100% compatible with the following Microsoft Office document formats : docx, pptx and xlsx.
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion

```

File: docs/OnlyOffice.rst [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+#############################
+OnlyOffice Connector for eXo
+#############################
+
+**What is OnlyOffice**
+
+`OnlyOffice Online Editors <https://www.onlyoffice.com/office-suite.aspx>`__ is a great open source solution 
+that lets you to create, edit  and collaborate online on your office 
+documents.
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
documents. OnlyOffice editors are 100% compatible with the following Microsoft Office document formats : docx, pptx and xlsx.
```

File: docs/OnlyOffice.rst [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+|image0|
+
+.. note:: OnlyOffice is 100% compatible with the following Microsoft Office document formats : docx, pptx and xlsx.
+
+eXo partnered with OnlyOffice to offer the enterprise-ready version of OnlyOffice editors to its customers along with the **commercial
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
.. note:: eXo partnered with OnlyOffice to offer the enterprise-ready version of OnlyOffice editors to its customers along with the **commercial editions of eXo Platform**. Thus, eXo customers are required to purchase a dedicated subscription to run the OnlyOffice connector add-on on their eXo Platform instance. However, a 30-day trial is prepackaged with the connector in order to test-drive the editors before purchase.
```

File: docs/OnlyOffice.rst [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+editions of eXo Platform**. Thus, eXo customers are required to purchase a
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion

```

File: docs/OnlyOffice.rst [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+dedicated subscription to run the OnlyOffice connector add-on on their eXo Platform instance.
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion

```

File: docs/OnlyOffice.rst [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+
+However, a 30-day trial is prepackaged. You can test-drive the connector by installing the add-on using this command:
```
Comment (plamarque) [inline docs/OnlyOffice.rst]:
```suggestion
To install the add-on, use the following command:
```

**Review by plamarque — CHANGES_REQUESTED** [review]
I've had the idea to put the warning about commercial editions as a note rather than the supported formats. @mzorai WDYT ?

**Comment by mzorai** [comment]
> I've had the idea to put the warning about commercial editions as a note rather than the supported formats. @mzorai WDYT ?

Yep that way it will be more visible to reader!

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**